v22.1/licensing-faqs.md

```diff
@@ -160,3 +160,11 @@ The CockroachDB team is committed to supporting the open-source community and wi
 ### Can I fork the CockroachDB project pre-BSL and create my own CockroachDB derivative with a different license?
 
 You can fork any historical version of CockroachDB in your own project, as allowed by the license available for that version, and modify it for your purpose. Note however that only the copyright holder (Cockroach Labs) can relicense the components that you forked from: your derivative will need to keep the original license at the time of the fork. Any component you copy from a BSL-licensed CockroachDB into your project will make the BSL apply to your project as well.
+
+### If Cockroach Labs is making available software to me under the Business Source License, does Cockroach Labs grant me any patent rights?
+
+The BSL does not explicitly reference patents in the text of the license.
+However, Cockroach believes that the BSL includes an implied patent license and
```

Comment:
Please change `Cockroach believes` to `Cockroach Labs believes`.

Reply:
done